apps/src/music/player/MusicLibrary.ts

```diff
@@ -201,7 +203,7 @@ export default class MusicLibrary {
     const folderId = id.substring(0, lastSlashIndex);
     const src = id.substring(lastSlashIndex + 1);
 
-    const folder = this.getFolderForFolderId(folderId);
+    const folder = this.getFolderForFolderId(folderId) as SoundFolderPack;
```

Comment:
Couple thoughts - _technically_, if the `id` happens to be an instrument, then this folder might be an instrument folder right? Should we maybe create a different lookup function like `getPackForPackId` that only performs the lookup on `this.packs`?

Another option might be to check this returned folder to confirm that it is a `SoundFolderPack`. One handy way to do this is to use [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) to both check and narrow the type at once. For example you might have a function like `isPack(folder: soundFolder): folder is SoundFolderPack { ... }` that you could use here. If you do something like
```
const folder = this.getFolderForFolderId(folderId);
if (!isPack(folder)) {
  // not a sound pack or invalid Id
  return null;
}
// Now, the type of folder has automatically been narrowed to SoundFolderPack because TS knows that the type predicate isPack() is true
```

The narrowing might not be that useful here, but could be useful in other parts of the code to avoid casting. There are a few examples of this in `ChordEvent` and `InstrumentEvent` that I added recently.